lib/generators/smart_answer/smart_answer_generator.rb

```diff
@@ -0,0 +1,43 @@
+class SmartAnswerGenerator < Rails::Generators::NamedBase
+  source_root File.expand_path("templates", __dir__)
+
+  def copy_flow
+    output_filename = "lib/smart_answer_flows/#{name.dasherize}.rb"
```

Comment:
Picky again - but there is only ever one file name that is parameterised in this file. So make the variable `filename`

Reply:
Thanks @reggieb - also makes the variable name shorter 👍